### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by rrbest** [ArkEcosystem/desktop-wallet#193]
The new release consumes more and more memory the longer it is left open.

**Comment by boldninja** [comment]
@luciorubeens Something to look at, I noticed same thing in Windows 8.1 x64.

**Comment by luciorubeens** [comment]
Yes, will investigate

**Comment by sleepdefic1t** [comment]
@luciorubeens 

There are quite a few comments on slack reporting these same issues.

@eugeneli /goldenpepe(slack) recently posted a screen reporting 'missing remote object'.
There are also Changer errors, which I've never NOT seen in the desktop-client.


There are several discussions around the web that may be relevant.
One possibly helpful discussion I found:   

[**Electron: Missing remote object #8205**](https://github.com/electron/electron/issues/8205)
-----

TL;DR
May be garbage collection issues.
Many commenting ditching ```remote``` in favor of ```ipcMain/ipcRenderer``` seems to work.

I'm not familiar with how it would be implemented,
but I hope it helps.

**Comment by fix** [comment]
my analysis is this line is making memory consumption:
https://github.com/ArkEcosystem/ark-desktop/blob/master/client/app/src/accounts/account.controller.js#L769

The issue is that `previousTx`array is hooked into an object and the whole array is never cleaned by the garbage collector of the javascript engine.

Since this is called every 8s here
https://github.com/ArkEcosystem/ark-desktop/blob/master/client/app/src/accounts/account.controller.js#L769

It gives you the idea how far it can goes.

**Comment by spresnal** [comment]
Is this still an issue? If so, can someone walk me through the steps to reproduce it? I have been unsuccessful in identifying anything using the dev tools memory profiler. I'm not sure if it's because my accounts don't have transactions or if it has to do something with delegates etc.

**Comment by boldninja** [comment]
@spresnal under number 2 its more details on this: https://blog.ark.io/new-ark-desktop-wallet-release-v1-4-0-2d4c286417f6 there is a temp solution atm if you are leaving it for longer periods of time you can switch to manual account refresh.
